phoenix-core/src/main/java/org/apache/phoenix/mapreduce/OrphanViewTool.java

```diff
@@ -445,7 +445,11 @@ private void removeLink(PhoenixConnection phoenixConnection, Key src, Key dst, P
                 COLUMN_NAME + (dst.getTenantId() == null ? " IS NULL" : " = '" + dst.getTenantId() + "'") + " AND " +
                 COLUMN_FAMILY + " = '" + (dst.getSchemaName() == null ? dst.getTableName() : dst.getSchemaName() + "." +
                 dst.getTableName()) + "'";
-        phoenixConnection.createStatement().execute(deleteQuery);
+        try {
+            phoenixConnection.prepareStatement(deleteQuery).execute();
+        } catch (SQLException e) {
```

Comment:
Why to handle the SQLException and make it IOException better to through the SQLException as it is?